clients/admin-ui/cypress/e2e/systems-plus.cy.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  stubPlus,
+  stubSystemCrud,
+  stubTaxonomyEntities,
+} from "cypress/support/stubs";
+
+describe("System management with Plus features", () => {
+  beforeEach(() => {
+    cy.login();
+    stubSystemCrud();
+    stubTaxonomyEntities();
+    stubPlus(true);
+    cy.intercept("GET", "/api/v1/system", {
+      fixture: "systems/systems.json",
+    }).as("getSystems");
+    cy.visit("/system");
+  });
+
+  describe("custom metadata", () => {
+    beforeEach(() => {
+      cy.intercept(
+        {
+          method: "GET",
+          pathname: "/api/v1/plus/custom-metadata/allow-list",
+          query: {
+            show_values: "true",
+          },
+        },
+        {
+          fixture: "taxonomy/custom-metadata/allow-list/list.json",
+        }
+      ).as("getAllowLists");
+      cy.intercept(
+        "GET",
+        `/api/v1/plus/custom-metadata/custom-field-definition/resource-type/*`,
+
+        {
+          fixture: "taxonomy/custom-metadata/custom-field-definition/list.json",
+        }
+      ).as("getCustomFieldDefinitions");
+      cy.intercept(
+        "GET",
+        `/api/v1/plus/custom-metadata/custom-field/resource/*`,
+        {
+          fixture: "taxonomy/custom-metadata/custom-field/list.json",
+        }
+      ).as("getCustomFields");
+    });
+
+    it("can populate initial custom metadata", () => {
+      cy.getByTestId("system-fidesctl_system").within(() => {
+        cy.getByTestId("more-btn").click();
+        cy.getByTestId("edit-btn").click();
+      });
+
+      // Should not be able to save while form is untouched
+      cy.getByTestId("save-btn").should("be.disabled");
+      const testId =
+        "input-customFieldValues.id-custom-field-definition-pokemon-party";
+      cy.getByTestId(testId).contains("Charmander");
+      cy.getByTestId(testId).contains("Eevee");
+      cy.getByTestId(testId).contains("Snorlax");
+      cy.getByTestId(testId).type("Bulbasaur{enter}");
+
+      // Should be able to save now that form is dirty
+      cy.getByTestId("save-btn").should("be.enabled");
```

Comment:
I'd recommend going a bit further here and clicking save in the test and asserting that the right API calls get made. There's a _lot_ of magic going on in `useCustomFields` that'd benefit from some coverage to keep an eye on it